maintenance/wikia/email.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Email
+ *
+ * This sends an email type (defined in extensions/wikia/Email) to the email address of your
+ * choice
+ *
+ */
+
+ini_set( 'display_errors', 'stderr' );
+ini_set( 'error_reporting', E_NOTICE );
+
+// Eliminate the need to set this on the command line
+if ( !getenv( 'SERVER_ID' ) ) {
+	putenv( "SERVER_ID=177" );
+}
+
+require_once( dirname( __FILE__ ) . '/../Maintenance.php' );
+
+/**
+ * Class Email\CLI
+ */
+class EmailCLI extends Maintenance {
+
+	protected $verbose = false;
+	protected $test = false;
+	protected $user = '';
+	protected $type = '';
+	protected $argString = '';
+	protected $list = false;
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Test sending email via the Email extension";
+		$this->addOption( 'test', 'Test mode; make no changes',
+			Maintenance::PARAM_OPTIONAL,
+			Maintenance::PARAM_IS_FLAG,
+			't'
+		);
+		$this->addOption( 'verbose', 'Show extra debugging output',
+			Maintenance::PARAM_OPTIONAL,
+			Maintenance::PARAM_IS_FLAG,
+			'v'
+		);
+
+		$this->addOption( 'user', 'Masquerade as this user when sending email. Can be username or ID.s',
```

Comment:
Is that trailing "s" a typo? I think this should be `Masquerade as this user when sending email. Can be username or ID`, yea?